src/main/cpp/t07_ip.cpp

```diff
@@ -29,10 +29,30 @@
 
 #include "t07_ip.h"
 #include <iostream>
+#include<string>
 
 
 using namespace std;
 
 int t07_ip() {
-
+string str,sub;
+int probel=0;
+bool flag=false;
+getline(cin,str);
+for(int i=0;i<str.length();i++)
+{
+  if(str[i]=='.'){
+    if(probel==0)sub=str.substr(probel,i-probel);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Condition 'probel==0' is always true](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513001)